apps/api/src/app/environments-v1/usecases/output-renderers/novu-branding-html.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * This is the HTML for the Novu branding image.
+ * Should be in par with the actual React component we show in the dashboard preview:
+ * @see apps/dashboard/src/components/workflow-editor/steps/email/novu-branding.tsx
+ */
+export const NOVU_BRANDING_HTML = `
+<table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="max-width:600px;min-width:300px;width:100%;margin:24px auto 8px auto;">
+  <tbody>
+    <tr style="width:100%">
+      <td align="center">
+        <img src="https://prod-novu-app-bucket.s3.us-east-1.amazonaws.com/assets/email-editor/powered-by-novu.png" alt="Powered by Novu" title="This email was sent using Novu - Open-source notification infrastructure" width="156" height="16" style="display:block;max-width:100%;height:auto;cursor:pointer;" data-novu-branding />
+      </td>
+    </tr>
+  </tbody>
+</table>
+`;
```

Comment:
Having the branding banner as image ended up being the most simple and reliable in email. SVGs gets sanitized, we can't reproduce the component using HTML cause email client doesn't have the styles and fonts etc etc.

Reply:
Seems to be working fine also when using HTML code blocks with entire templates etc.